.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ references:
   #
   # Workspace
   #
-  container_config_node8: &container_config_node8
+  container_config_node: &container_config_node
```

Comment:
Do we need to specify version 10 with these (and the others in this file) if we have specified `- image: circleci/node:10.13`? E.g.

`container_config_node10: &container_config_node10`

It might also be worth changing `- image: circleci/node:10.13` to `- image: circleci/node:10.x`.

Reply:
It seemed to be that `container_config_node` was just a name to reference the workspace and only the image number mattered. I'm new to CircleCI though so I could be wrong.

Reply:
This is easily possible, which makes we wonder why we include the version number elsewhere… 🤔

Reply:
I'll update to follow the convention.